client/src/Movies/Movie.js

```diff
@@ -45,7 +46,13 @@ export default class Movie extends Component {
     const { title, director, metascore, stars } = this.state.movie;
     return (
       <div className="save-wrapper">
-        <div className="movie-card">
+        <MovieCard 
+          title={title}
+          director={director}
+          metascore={metascore}
+          stars={stars}  
```

Comment:
thank you for separating these out onto other lines. it's so nice to read! you could also have passed the whole movie object here and destructured inside of movie card instead.